block/README.md

```diff
@@ -0,0 +1,46 @@
+# Metrics Policy
+
+Use the Metrics Policy as an example of how to create a task that is executed periodically in a single worker.
```

Comment:
`Block Policy`. I would also add a brief explanation on what the policy is looking to achieve and how it uses the timer for that purpose

Reply:
And config explaining